custom_components/sat/simulator/__init__.py

```
@@ -84,7 +85,7 @@ async def async_set_control_max_setpoint(self, value: float) -> None:
         self._maximum_setpoint = value
         await super().async_set_control_max_setpoint(value)
 
-    async def async_control_heating_loop(self, climate: Optional[SatClimate] = None, pwm_state: Optional[PWMStatus] = None, _time=None) -> None:
+    async def async_control_heating_loop(self, timestamp: float = None, climate: Optional[SatClimate] = None, pwm_state: Optional[PWMState] = None) -> None:
```

Ruff (0.14.10)

Unused method argument: `timestamp`

(ARG002)